doc/nrf/working_with_nrf/nrf70/features.rst

```diff
@@ -43,7 +43,9 @@ Devices in the nRF70 Series are supported by the following boards in the `Zephyr
    * - nRF7002 DK
      - PCA10143
      - ``nrf7002dk_nrf5340_cpuapp``
-     -
+     - | `Product Specification <Product specification for nRF70 Series devices_>`_
+       | :ref:`Getting started <nrf7002dk_nrf5340>`
```

Comment:
Note, this doc is already linked just above the diagram. We could drop this link here, leaving just the two infocenter ones.